apps/nextjs-website/src/helpers/chatbot.helper.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+import { useEffect, useState } from 'react';
+import { getChatbotQueries, sendChatbotQuery } from '@/lib/chatbot';
+import { ChatbotQueries } from '@/lib/chatbot/queries';
+
+export const useChatbot = () => {
+  const [isLoaded, setIsLoaded] = useState(false);
```

Comment:
if this `isLoaded` value is defined only by `sessionId` presence, i think is better and safer to remove this state and simpy add an isLoaded attribute as a return of this hook.
Something like this:
```
  return {
    isLoaded = !!sessionId,
    isAwaitingResponse,
    queries,
    sendQuery,
  };
```

Reply:
The isLoaded state changes only after the chat messages are fetched or an error during fetch occurs